CHANGELOG.md

```diff
@@ -1,4 +1,52 @@
+- Added: new `Lucky::CatchUnpermittedAttribute` mixin for `Shared::Field` component. [#1052](https://github.com/luckyframework/lucky/pull/1052)
+- Added: new methods in Actions for accessing params from different sources like `from_json`, `from_query`, `from_form`, and `from_multipart`. [#1053](https://github.com/luckyframework/lucky/pull/1053)
+- Updated: generated pages to have some default text pointing to the location of the file to edit. [#1057](https://github.com/luckyframework/lucky/pull/1057)
+- Fixed: incorrect pluralization of resources on `NewPage`. [#1058](https://github.com/luckyframework/lucky/pull/1058)
+- Updated: all action "callbacks" are officially named "pipes". All pipes only log when halted by default. [#1062](https://github.com/luckyframework/lucky/pull/1062)
+- Updated: the `lucky dev` watcher stop telling you when you changed a file because you know you just changed that file. [#1065](https://github.com/luckyframework/lucky/pull/1065)
+- Added: a new HTTP handler to set the `request.remote_address` if the `X-Forwarded-For` header is set. [#1059](https://github.com/luckyframework/lucky/pull/1059)
+- Added: ability to set custom directory when generating a new Lucky project [See LuckyCli](https://github.com/luckyframework/lucky_cli/pull/464)
+- Added: ability to set your postgres DB port with ENV var. [See LuckyCli](https://github.com/luckyframework/lucky_cli/pull/469)
+- Added: a `robots.txt` file to generated web apps by default. [See LuckyCli](https://github.com/luckyframework/lucky_cli/pull/472)
+- Added: new compiling spinner graphic for a cleaner UX. [See LuckyCli](https://github.com/luckyframework/lucky_cli/pull/481)
+- Updated: some comments on the generated main app file. [See LuckyCli](https://github.com/luckyframework/lucky_cli/pull/484)
+- Added: lots of internal documentation. (many small commits to LuckyCli)
+- Updated: generated `UserSerializer` to inherit from `BaseSerializer`. [See LuckyCli](https://github.com/luckyframework/lucky_cli/pull/489)
+- Updated: cookies to default to `http_only`. [See LuckyCli](https://github.com/luckyframework/lucky_cli/pull/491)
+- Updated: node dependencies in generated web apps. [See LuckyCli](https://github.com/luckyframework/lucky_cli/pull/493)
+- Added: new `system_check` script along with some refactors to make checking that your app is setup a lot easier. [See LuckyCli](https://github.com/luckyframework/lucky_cli/pull/482)
+- Removed: ability to pass a raw hash to an `Avram::SaveOperation`. [See Avram](https://github.com/luckyframework/avram/pull/312)
+- Added: ability to `skip_schema_enforcer` for certain models. [See Avram](https://github.com/luckyframework/avram/pull/314)
+- Added: setting local config files for development that won't be added to your project's repo. [See Avram](https://github.com/luckyframework/avram/pull/314)
```

Comment:
The PR link is incorrect and I don't remember which PR this is referring too. I already forgot what happened here :P

Reply:
It technically is correct, but you snuck in a feature 😛  
* https://github.com/luckyframework/avram/pull/314/files#diff-a084b794bc0759e7a6b77810e01874f2R6
* https://github.com/luckyframework/avram/pull/314/files#diff-95ff4954c90140a5899f092df800b0f3R3

I could link to those two lines, but I figured meh, let this one slide. I don't think I caught that when you submitted it originally 😅

Reply:
Oh I see. Those are just for local development with Avram and should not affect anyone else, so I think this can be removed. It's only useful for local dev of Avram so you can add `config/database.local.cr` with committing it

Reply:
ah! ok.